### ISSUE TITLE
Article 31 enhancements

### ISSUE THREAD

**Issue by ivanmladek** [liberland/Constitution#599]
Consoildate article 31 comments

**Comment by ivanmladek** [comment]
Right off the bat I recommend REMOVING the qualifier "fit for personal defense." This opens the door for gun regulation. Suggest ADDING "The State shall not require any registrations or qualifications for arm ownership and purchase and manufacture and furthermore no arm or arm bearer registry shall be established, being contrary to the Right to Privacy"

**Comment by UpLib** [comment]
I agree that _"fit for personal defense"_ could become a pandora's box.
But, removing it puts practically no limit to the arms we could bring to LL.
eg if my LL neighbour brings a biological arm ... I would feel unsafe.
There are probably some reasonable limits to be fixed.
Maybe the limits could be defined by a referendum ?
